src/ReturnBookUi.java

```diff
@@ -25,25 +50,26 @@ public void run() {
                     break;
                 }
                 case READY: {
-                    String bookStr = input("Scan Book (<enter> completes): ");
-                    if (bookStr.length() == 0) {
+                    String bookIdString = getInput("Scan Book (<enter> completes): ");
+                    if (bookIdString.length() == 0) {
                         control.scanningComplete();
                     } 
                     else {
                         try {
-                            int bookId = Integer.valueOf(bookStr).intValue();
+                            int bookId = Integer.valueOf(bookIdString).intValue();
                             control.bookScanned(bookId);
                         } 
-                        catch (NumberFormatException e) {
-                            output("Invalid bookId");
+                        catch (NumberFormatException error) {
+                            output("Invalid Book Id");
                         }
                     }
                     break;
                 }
+                
                 case INSPECTING: {
-                    String ans = input("Is book damaged? (Y/N): ");
+                    String answer = getInput("Is book damaged? (Y/N): ");
                     boolean isDamaged = false;
```

Comment:
Add a new line in between lines 71 and 72 to make spacing uniform throughout file.